notebooks/tutorials/hello-syft/01-hello-syft.ipynb

```diff
@@ -11,6 +11,45 @@
     "# Hello Syft"
```

Comment:
<p>use </p><pre class="ql-syntax" spellcheck="false">request = project.requests[0]
</pre><p> </p><p>and</p><pre class="ql-syntax" spellcheck="false">func = request.changes[0].link
func
</pre><p>and</p><pre class="ql-syntax" spellcheck="false">print(func.code)
</pre><p>like in the 0.8 notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/7775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>How to get the <code>project</code>?</p>

Reply:
<p>It seems that <code>root_domain_client.projects</code> is empty because <code>project.create_code_request(get_mean_age, external_ds)</code> return <code>SyftError: Duplication Key Error: syft.service.code.user_code.UserCode</code> even after restarting the kernel and running the whole notebook.   </p>